ThunderCloud/StormTableRow.swift

```diff
@@ -34,25 +34,6 @@ class StormTableRow: TableRow {
 				return
 			}
 			
-			//TODO: Make sure this still works!
```

Comment:
This was for RTL languages I believe, do we need to make sure they are still supported in Storm?